packages/e2e/artillery/StakePoolSearch.yml

```diff
@@ -0,0 +1,25 @@
+config:
+  target: 'http://localhost:4000'
```

Comment:
For other reviewers, there's a CLI override argument: https://www.artillery.io/docs/guides/guides/command-line#override-the-target-url

Reply:
For me too! I didn't noticed it, thank you @rhyslbw